apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedissonPluginConfig.java

```diff
@@ -20,14 +20,18 @@
 
 import org.apache.skywalking.apm.agent.core.boot.PluginConfig;
 
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
 public class RedissonPluginConfig {
     public static class Plugin {
         @PluginConfig(root = RedissonPluginConfig.class)
         public static class Redisson {
             /**
              * If set to true, the parameters of the Redis command would be collected.
              */
-            public static boolean TRACE_REDIS_PARAMETERS = false;
+            public static boolean TRACE_REDIS_PARAMETERS = true;
```

Comment:
Please don't change the default value.

Reply:
If set to false, can I still get "cache. key"?
This value is true in 'org.apache.skywalking.apm.plugin.jedis.v3.JedisPluginConfig'

Reply:
> If set to false, can I still get "cache. key"? This value is true in 'org.apache.skywalking.apm.plugin.jedis.v3.JedisPluginConfig'

I probably understand, the default value does not need to be modified, sorry.